app/views/spree/admin/adjustments/edit.html.haml

```diff
@@ -1,4 +1,4 @@
-= render :partial => 'spree/admin/shared/order_tabs', :locals => { :current => 'Adjustments' }
+= render :partial => 'spree/admin/shared/order_page_title'
```

Comment:
same here, I have commented before that you need the call to order tabs here and you have added it. But now it's gone again.